Defs/Ammo/Advanced/PlasmaGasCartridge.xml

```diff
@@ -0,0 +1,469 @@
+    <description>Craft 100 Plasma Gas Cartridges .</description>
+    <jobString>Making Plasma Gas Cartridges .</jobString>
+    <ingredients>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Plasteel</li>
+          </thingDefs>
+        </filter>
+        <count>3</count>
+      </li>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>ComponentIndustrial</li>
+          </thingDefs>
+        </filter>
+        <count>3</count>
+      </li>
+    </ingredients>
+    <fixedIngredientFilter>
+      <thingDefs>
+        <li>Plasteel</li>
+        <li>ComponentIndustrial</li>
+      </thingDefs>
+    </fixedIngredientFilter>
+    <products>
+      <Ammo_SWPlasmaGasCartridge>100</Ammo_SWPlasmaGasCartridge>
+    </products>
+  </RecipeDef>
+
+  <RecipeDef ParentName="ChargeAmmoRecipeBase">
+    <defName>MakeAmmo_SWPlasmaGasCartridge_AP</defName>
+    <label>make Plasma Gas Cartridges (Conc.)  x100</label>
+    <description>Craft 100 Plasma Gas Cartridges (Conc.) .</description>
+    <jobString>Making Plasma Gas Cartridges (Conc.) .</jobString>
+    <ingredients>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Plasteel</li>
+          </thingDefs>
+        </filter>
+        <count>3</count>
+      </li>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>ComponentIndustrial</li>
+          </thingDefs>
+        </filter>
+        <count>1</count>
+      </li>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Uranium</li>
+          </thingDefs>
+        </filter>
+        <count>3</count>
+      </li>
+    </ingredients>
+    <fixedIngredientFilter>
+      <thingDefs>
+        <li>Plasteel</li>
+        <li>ComponentIndustrial</li>
+      </thingDefs>
+    </fixedIngredientFilter>
+    <products>
+      <Ammo_SWPlasmaGasCartridge_AP>100</Ammo_SWPlasmaGasCartridge_AP>
+    </products>
+  </RecipeDef>
+
+
+  <RecipeDef ParentName="ChargeAmmoRecipeBase">
+    <defName>MakeAmmo_SWPlasmaGasCartridge_Ion</defName>
+    <label>make Plasma Gas Cartridges (Ion)  x100</label>
+    <description>Craft 100 SWPlasmaGas Cartridges (Ion) .</description>
+    <jobString>Making Plasma Gas Cartridges (Ion) .</jobString>
+    <ingredients>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>Plasteel</li>
+          </thingDefs>
+        </filter>
+        <count>6</count>
+      </li>
+      <li>
+        <filter>
+          <thingDefs>
+            <li>ComponentIndustrial</li>
+          </thingDefs>
+        </filter>
+        <count>3</count>
+      </li>
+    </ingredients>
+    <fixedIngredientFilter>
+      <thingDefs>
+        <li>Plasteel</li>
+        <li>ComponentIndustrial</li>
+      </thingDefs>
+    </fixedIngredientFilter>
+    <products>
+      <Ammo_SWPlasmaGasCartridge_Ion>100</Ammo_SWPlasmaGasCartridge_Ion>
+    </products>
+  </RecipeDef>
+  
+  
+  <!-- Damage and Hediff Defs -->
+  
+  
+  <DamageDef>

```

Comment:
Damage and Hediff defs should also be put in separate XML files in the Star Wars patch folder.

Reply:
Done.